package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@department-of-veterans-affairs/component-library-monorepo",
   "description": "The monorepo containing the component library packages.",
   "private": true,
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
I think you intended to increment the `packages/core/` version number. This version number is not in use afik.

Reply:
Oh, sheesh.

Reply:
Reverted. Do I need to update the core version number if I didn't change anything in the core package?

Reply:
We use the core version for [releases](https://github.com/department-of-veterans-affairs/component-library#releasing) so it should be incremented if we are publishing the NPM package. I think generally we just bump core in the last PR that will be included in the release: https://github.com/department-of-veterans-affairs/component-library/releases